tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "include": ["./packages/*/src/**/*.ts"],
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "lib": [
+      "esnext"
+    ],
+    "noEmit": true,
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "skipLibCheck": true,
+    "resolveJsonModule": true,
+    "paths": {
```

Comment:
fixed the duplicated `compilerOptions` in https://github.com/babel/babel-polyfills/pull/118#issuecomment-1073346266